test/spec/modules/yieldmoBidAdapter_spec.js

```diff
@@ -218,7 +218,7 @@ describe('YieldmoAdapter', function () {
 
       it('should add pubcid as parameter of request', function () {
         const pubcid = 'c604130c-0144-4b63-9bf2-c2bd8c8d86da2';
-        const pubcidBid = mockBannerBid({crumbs: undefined, userId: {pubcid}});
+        const pubcidBid = mockBannerBid({crumbs: undefined, userIdAsEids: {pubcid}});
```

Comment:
several of the changes here are incorrect for the same reason the yieldmo `getId` change was - at runtime, `userIdAsEids` is never a map.